evalml/tests/component_tests/test_lgbm_classifier.py

```diff
@@ -281,3 +282,38 @@ def test_binary_label_encoding(mock_fit, mock_predict, X_y_binary):
     assert_series_equal(y_arg, y_numeric)
 
     clf.predict(X)
+
+
+def test_binary_rf_not_defaults(X_y_binary):
```

Comment:
Do we have coverage somewhere for calling `fit` with the default parameters? If I follow correctly, that was one of the cases which was erroring. Specifically, with `boosting_type="rf"` and `bagging_freq=0`